### ISSUE TITLE
PAE fermata over barline

### ISSUE THREAD

**Issue by craigsapp** [rism-ch/verovio#1128]
Is there a way of showing a fermata over a barline in PAE?

It does not seem possible:

<img width="710" alt="Screen Shot 2019-07-25 at 2 01 06 PM" src="https://user-images.githubusercontent.com/3487289/61872811-9067a980-aee4-11e9-9a4c-074d039d16fa.png">

<img width="714" alt="Screen Shot 2019-07-25 at 2 01 19 PM" src="https://user-images.githubusercontent.com/3487289/61872820-93fb3080-aee4-11e9-91a7-75f78266d1eb.png">

<img width="721" alt="Screen Shot 2019-07-25 at 2 01 28 PM" src="https://user-images.githubusercontent.com/3487289/61872830-99587b00-aee4-11e9-9ddf-1b6e4b0b6cac.png">

<img width="702" alt="Screen Shot 2019-07-25 at 2 03 21 PM" src="https://user-images.githubusercontent.com/3487289/61872865-b1c89580-aee4-11e9-8c51-39878fd57498.png">

Or is there a bug in the PAE importer?

**Comment by rettinghaus** [comment]
> include only one note or rest

https://www.iaml.info/plaine-easie-code

**Comment by craigsapp** [comment]
Yes, the person who asked me about it had a print-out of that page, and I also had checked the same online page.  

> include only one note or rest

That can be interpreted as: you can only include one note or rest in the parentheses, meaning that you can not include two or more notes/rests within a parentheses.  Or it can be interpreted that only rests and notes can be placed in parentheses, meaning nothing else can be placed in parentheses AND that only one note/rest can be placed inside a parentheses.  It does not explicitly forbid putting a measure within parentheses depending on which interpretation you want to use 😄 

If invalid, then the above graphical examples are not handled correctly: the parentheses are around the barline and not the note, so there should not be a fermata on the note...

In any case the encoder of the incipit will omit the fermata on the barline when submitting to RISM.

This issue can either be closed or can be converted into an enhancement of the PAE converter to allow for fermatas on barlines and/or to not put them on the note after the barline.

**Comment by rettinghaus** [comment]
Although it is not Verovio's job to fix encoding errors, I consider your test cases indeed buggy. 
I fixed this with 90b741350c17b247ef25c99f7b6494bce30754af

```
@clef:G-2
@keysig:0
@timesig:4/4
@data:'4x()CD(b)E(F)+(/)(FxGA)(-)B(+)/1B//
```
![fermata_pae_001(1)](https://user-images.githubusercontent.com/7693447/61997817-b5f0e080-b0a7-11e9-846c-96f1b2c7f906.png)

**Comment by craigsapp** [comment]
Suppression of false fermatas implemented with pull request https://github.com/rism-ch/verovio/pull/1131
